Source/Core/Common/GL/GLInterface/AGL.mm

```diff
@@ -49,15 +33,34 @@
     return false;
   }
 
-  if (cocoaWin == nil)
+  if (window_handle)
```

Comment:
Yeah, the general style is to situate early-return cases first. It makes it much nicer to read (conceptually) as it places invalid code and any necessary handling first before actual behavior. If an indentation makes code nicer to read, that's usually a signifier that the contained code should be moved to another function, or it's too intermixed with other stuff.